packages/components/src/palette-edit/index.tsx

```diff
@@ -164,6 +208,7 @@ function Option( {
 				<FlexItem>
 					<IndicatorStyled
 						style={ { background: value, color: 'transparent' } }
+						isSelected={ isEditing }
```

Comment:
What do you think about this?

`IndictorStyled` now requires an `isSelected` prop:

<details
<summary>TS error before this change</summary>

```sh
packages/components/src/palette-edit/index.tsx:209:7 - error TS2741: Property 'isSelected' is missing in type '{ style: { background: string; color: "transparent"; }; }' but required in type '{ className?: string | undefined; tooltipText?: string | undefined; isSelected: boolean; selectedIconProps?: Omit<{ icon: Element; size?: number | undefined; } & { isPressed?: boolean | undefined; } & Pick<...>, "size" | "icon"> | undefined; }'.

209      <IndicatorStyled
          ~~~~~~~~~~~~~~~

  packages/components/src/circular-option-picker/types.ts:60:2
    60  isSelected: boolean;
        ~~~~~~~~~~
    'isSelected' is declared here.


Found 1 error.
```
</details>

Reply:
I might be missing something, but is this supposed to change anything visually? If anything, think it might be better to move this to a separate issue. For now I think it makes sense to change the CircularOptionPicker `isSelected` prop to be optional.

Reply:
No, it shouldn't change anything visually.

Good idea.

https://github.com/WordPress/gutenberg/pull/47764/commits/fcf7d477154be20da197e22a4e6c23cceb9cf590 changes the `CircularOptionPicker.Option` `isSelected` prop to optional.